api/controllers/MerchStoreController.ts

```diff
@@ -177,6 +179,21 @@ export class MerchStoreController {
     return { error: null, order };
   }
 
+  @Post('/order/verification')
+  async verifyMerchOrder(@Body() verifyOrderRequest: VerifyMerchOrderRequest,
```

Comment:
We changed it to its own type instead of `ApiResponse`, Don't know if having the boolean would be needed though since an error would be thrown in any event there is a valid order